.github/workflows/sync-labels.yaml

```diff
@@ -0,0 +1,32 @@
+# Install this in .github/workflows/ to automate label updates
+name: Sync Labels
+
+on:
+  workflow_dispatch:
+    # Allows for manual triggering.
+    inputs:
+      dry_run:
+        description: "If true, don't make any actual changes"
+        required: False
```

Comment:
This probably doesn't really matter, but why is this "false" capitalized when the other one isn't?

Reply:
Good question.  For a lot of actions that already exist, I'm copying the installation instructions or workflow templates from their documentation, or combining sections of other workflows.  I seem to have copied `required` from another workflow, but written `default` from scratch and used lowercase.

GitHub Actions docs seem to indicate that case doesn't matter.  I'll normalize them all to lowercase.